test/utils/FakeObjects.scala

```diff
@@ -46,6 +46,121 @@ import models.properties.AllowedAction.{CHECK, VIEW_DETAILED_VALUATION}
 
 trait FakeObjects {
 
```

Comment:
Hard to tell what/if anything has changed here as it seems to have all moved about but tests are passing so happy days 👍